extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -271,18 +280,83 @@ void ConsumeWindowsEventLog::onTrigger(const std::shared_ptr<core::ProcessContex
     return;
   }
 
-  const auto flowFileCount = processQueue(session);
+  const auto commitAndSaveBookmark = [&] (const std::wstring& bookmarkXml){
+    const auto before_commit = std::chrono::steady_clock::now();
+    session->commit();
+    logger_->log_debug("processQueue commit took %llu ms",
+                       std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::steady_clock::now() - before_commit).count());
```

Comment:
There's already a thread about `%llu`, but this case is somewhat different. Here, the standard doesn't specify the returned type, but it specifies the minimum number of bits.
I suggest assigning the result of `milliseconds::count` to a `uint64_t` (where it will definitely fit), then printing it as a normal `uint64_t`, i.e. with the `"%" PRIu64` format specifier

Reply:
Fixed.